tests/unit/test_merger.py

```diff
@@ -272,3 +272,55 @@ def test_real_record_merge_regression_1_authors_mismatch_on_update():
     assert sorted(conflict, key=itemgetter("path")) == sorted(
         expected_conflicts, key=itemgetter("path")
     )
+
+
+def test_titles_change():
+    root = {}
+
+    head = {"titles": [{"source": 1, "title": "Title 1"}, {"source": 2, "title": "Title 2"}]}
+
+    update = {"titles": [{"source": 3, "title": "Title 3"}]}
+
+    merged, conflict = merge(root, head, update)
+    expected_conflicts = [{'path': '/titles/0', 'op': 'remove', 'value': {'title': 'Title 3', 'source': 3}, '$type': 'REMOVE_FIELD'}]
```

Comment:
sorry I didn't see this earlier, but I think this should be an addition conflict, because otherwise rejecting it won't bring you back to the previous state of the record (which is the property we want).

Reply:
I had to add new conflict type in json-merger as urrent conflicts generates `add` operation (MANUAL_MERGE and ADD_BACK_TO_HEAD) in json patch are generating path with `/-` at the end which will result in adding element at the end of the list. I also modified json-merger to generate INSERT instead of REMOVE conflicts